.vscode/snippets.code-snippets

```diff
@@ -0,0 +1,30 @@
+{
+	// Place your curriculum workspace snippets here. Each snippet is defined under a snippet name and has a scope, prefix, body and 
+	// description. Add comma separated ids of the languages where the snippet is applicable in the scope field. If scope 
+	// is left empty or omitted, the snippet gets applied to all languages. The prefix is what is 
+	// used to trigger the snippet and the body will be expanded and inserted. Possible variables are: 
+	// $1, $2 for tab stops, $0 for the final cursor position, and ${1:label}, ${2:another} for placeholders. 
+	// Placeholders with the same ids are connected.
+	// Example:
+	// "Print to console": {
+	// 	"scope": "javascript,typescript",
+	// 	"prefix": "log",
+	// 	"body": [
+	// 		"console.log('$1');",
+	// 		"$2"
+	// 	],
+	// 	"description": "Log output to console"
+	// }
+	"force markdown": {
+		"prefix": "fmd",
+		"body": [
+			"<!-- livebook:{\"force_markdown\":true} -->"
+		]
+	},
+	"break markdown": {
+		"prefix": "bmd",
+		"body": [
+			"<!-- livebook:{\"break_markdown\":true} -->"
+		]
+	}
+}
```

Comment:
I've purposely decided to commit these snippets, though I'm not sure I should.

I find these extremely useful for working with livebook and believe it demonstrates a good example.